problem_2/main.ipynb

```diff
@@ -27,6 +27,7 @@
     "import pandas as pd\n",
```

Comment:
<p>This addition of pivoting the tags was a good choice! Good work! :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeanMarcusJensen/ACIT4610_Portifolio_KMHS/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>